hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -139,8 +139,6 @@ public void loadAllFromStore(List<Data> keys) {
             Future f = recordStoreLoader.loadValues(keys);
             loadingFutures.add(f);
         }
-
-        keyLoader.trackLoading(false, null);
     }
```

Comment:
I thought it would be too easy! thanks! what would you suggest to do? to split these paths?